portfolio/src/main/webapp/script.js

```diff
@@ -2,9 +2,17 @@ function changeImage() {
   document.getElementById("switch").className = "index-image-2"; 
 }
 
-async function populateGreeting() {
-  const response = await fetch('/data');
-  const greeting = await response.text();
-  document.getElementById("greeting").innerText = greeting;
+function createListElement(text) {
+  const listElement = document.createElement('li');
+  listElement.innerText = text;
+  return listElement;
 }
-populateGreeting();
\ No newline at end of file
+
+function populateComments() {
```

Comment:
nit: add JSDoc comment to these functions.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments